TU-Agent/buildConstruction.test2g

```diff
@@ -3,18 +3,24 @@ use agentKnowledge as knowledge.
 use agentGoals as goals.
 use buildConstruction as module.
 
-test tygronagent with
+test buildConstruction with
 	%check if there is at least an oldbuilding believe.
-	pre {true}
+	pre { bel(availableLandPolygon(MP)) }
```

Comment:
The module can still run when theres no such believe. See the code in the module.